google/cloud/bigquery/v2/minimal/internal/json_utils.cc

```diff
@@ -93,6 +93,19 @@ void ToJson(std::chrono::system_clock::time_point const& field,
   j[name] = std::to_string(m);
 }
 
+void removeEmptyArraysAndObjects(nlohmann::json& j) {
```

Comment:
The Google Style Guide requires this to be `RemoveEmptyArrayAndObjects` (note the initial uppercase).